ts/sagas/profile.ts

```diff
@@ -345,6 +347,17 @@ function* handleLoadBonusBeforeRemoveAccount() {
     // they will be loaded individually
     yield put(loadAllBonusActivations.request());
   }
+
+  const cgnActive: ReturnType<typeof cgnDetailSelector> = yield select(
+    cgnDetailSelector
+  );
+
+  if (pot.isNone(cgnActive)) {
```

Comment:
As improvement, what do you think if CGN is active as program, by consulting the remote feature flag (`isCGNEnabledSelector`)?

Reply:
Yep nice idea, added in db273e4